[2024-05-10T17:15:23.204+0700] {processor.py:161} INFO - Started process (PID=81178) to work on /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:15:23.205+0700] {processor.py:830} INFO - Processing file /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-10T17:15:23.205+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:15:23.205+0700] {dagbag.py:545} INFO - Filling up the DagBag from /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:15:23.225+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:15:23.225+0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-10T17:15:23.226+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:15:23.226+0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-10T17:15:23.226+0700] {processor.py:842} WARNING - No viable dags retrieved from /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:15:23.266+0700] {processor.py:183} INFO - Processing /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2024-05-10T17:16:57.292+0700] {processor.py:161} INFO - Started process (PID=81919) to work on /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:16:57.293+0700] {processor.py:830} INFO - Processing file /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-10T17:16:57.293+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:16:57.293+0700] {dagbag.py:545} INFO - Filling up the DagBag from /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:16:57.315+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:16:57.314+0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-10T17:16:57.315+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:16:57.315+0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-10T17:16:57.315+0700] {processor.py:842} WARNING - No viable dags retrieved from /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:16:57.354+0700] {processor.py:183} INFO - Processing /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2024-05-10T17:19:07.221+0700] {processor.py:161} INFO - Started process (PID=82966) to work on /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:19:07.223+0700] {processor.py:830} INFO - Processing file /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-10T17:19:07.223+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:19:07.223+0700] {dagbag.py:545} INFO - Filling up the DagBag from /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:19:07.246+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:19:07.246+0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-10T17:19:07.247+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:19:07.247+0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-10T17:19:07.247+0700] {processor.py:842} WARNING - No viable dags retrieved from /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:19:07.289+0700] {processor.py:183} INFO - Processing /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-05-10T17:21:16.458+0700] {processor.py:161} INFO - Started process (PID=84254) to work on /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:21:16.459+0700] {processor.py:830} INFO - Processing file /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-05-10T17:21:16.459+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:21:16.459+0700] {dagbag.py:545} INFO - Filling up the DagBag from /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:21:16.481+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:21:16.480+0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-05-10T17:21:16.481+0700] {logging_mixin.py:188} INFO - [2024-05-10T17:21:16.481+0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-05-10T17:21:16.481+0700] {processor.py:842} WARNING - No viable dags retrieved from /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-05-10T17:21:16.521+0700] {processor.py:183} INFO - Processing /opt/anaconda3/envs/kafka_project/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
